### PR TITLE
OHRI-966 OHRI-967 Feature/ohri 511 care and treatment s27 d20

### PR DESCRIPTION
**Pull request by larslemos**
This PR has the place Holders for the following Scope:

image

### CODE REVIEW

**Comment by larslemos** [comment]
@pirupius can you adjust the style for the columns and rows?

File: packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+      {isLoading ? (
+        <CodeSnippetSkeleton type="multi" />
+      ) : allRows.length > 0 ? (
+        <Tile>
+          <div className={styles.currentARvRegimenContainer}>
+            <div className={styles.currentARvRegimenHeader}>
+              <h4 className={styles.title}> {headerTitle} </h4>
+            </div>
+
+            {columns.map(column => (
```
Comment (ebambo) [inline packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx:159]: @larslemos this is why you don't get the tiles side by side. You need to wrap the columns in a row. Look at the tiles we have on the dashboard. Please change this

File: packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx [hunk 2]
```diff
@@ -1,20 +1,140 @@
...
-import { EmptyStateComingSoon } from 'openmrs-esm-ohri-commons-lib';
+import {
+  EncounterTile,
+  EncounterTileColumn,
+  findObs,
+  getEncounterValues,
+  getObsFromEncounter,
+} from '../../../../../esm-commons-lib/src/components/encounter-tile/encounter-tile.component';
 
+// import {
```
Comment (ebambo) [inline packages/esm-hiv-app/src/views/service-summary/encounter-list/service-summary-encounter-list.component.tsx:12]: @larslemos why are you pushing comments?

**Review by ebambo — CHANGES_REQUESTED** [review]

File: packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx (hunk 1 above)
Comment (larslemos) [reply packages/esm-commons-lib/src/components/encounter-tile/encounter-tile.component.tsx:158]: DONE

**Comment by larslemos** [comment]
As It looks now.
Screen Shot 2022-08-10 at 02 07 04

**Pull request merged**